src/main/java/com/example/demo/CardData/CardDataService.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.CardData;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CardDataService {
+
+    @Autowired
+    public CardDataService() {
+
+    }
+
+    public List<CardData> getCardData() {return List.of();}
```

Comment:
Shouldn't we get data from reporsitory? List.of() just return empty list right?

Reply:
done